docs/CHANGELOG.md

```diff
@@ -9,7 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [dev]
 
-...
+### Added
+
+- Gas bumping logic to the `SuggestedPriceEstimator`
```

Comment:
Can you add a few more details to this? Perhaps explain that gas bumping in this case means re-fetching a new price and on top of that you can now add a buffer.

Reply:
Sure thing! Just updated